Sources/Intermodular/Helpers/UIKit/UIHostingMotionEffectsView.swift

```diff
@@ -0,0 +1,85 @@
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+import UIKit
+import SwiftUI
+
+/// Wrapper that applies a motion effect to an underlying hosting controller, 
+/// whose view will a `SwiftUI.View` (the `rootView` passed in at init time).
+open class UIHostingMotionEffectsView<Content: View>: UIView {
```

Comment:
I wonder if this is more accurately named as "UIHostingInterpolatingMotionEffectsView"